CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/RunPane.java

```diff
@@ -305,6 +299,14 @@ public RunPane(final ImportController importController, final String displayText
         });
     }
 
+    private void setFilterStyle(final String value) {
+        if (setFilter(value)) {
+            filterField.setStyle(FILTER_STYLE);
+        } else {
+            filterField.setStyle(FILTER_STYLE_ALERT);
+        }
```

Comment:
noting you just copied across what was already there, are you able to convert this to a ternary statement?
`filterField.setStyle(setFilter(value) ? FILTER_STYLE : FILTER_STYLE_ALERT)`